packages/router-plugin/src/webpack.ts

```diff
@@ -0,0 +1,62 @@
+import { createWebpackPlugin } from 'unplugin'
+import { unpluginRouterCodeSplitterFactory } from './code-splitter'
+import { configSchema } from './config'
+import { unpluginRouterGeneratorFactory } from './router-generator'
+import { unpluginRouterComposedFactory } from './composed'
+import type { Config } from './config'
+
+/**
+ * @example
+ * ```ts
+ * export default {
+ *   // ...
+ *   plugins: [TanStackRouterGeneratorWebpack()],
+ * }
+ * ```
+ */
+const TanStackRouterGeneratorWebpack = /* #__PURE__ */ createWebpackPlugin(
+  unpluginRouterGeneratorFactory,
+)
+
+/**
+ * @experimental Do not use this plugin yet
+ *
+ * Unplugin's Webpack integration doesn't support the `resolveId` and `transform` hooks.
```

Comment:
@SeanCassiere we can remove this comment but still leave the feature as experimental and unstable, right?

Reply:
Yup you are right.